### PR TITLE
Mention "luca" in `check_in_qr_code` (Addresses #1461)

### LINKED ISSUE #1461: FAQ does not mention Luca QR codes
According to several sources CWA is supposed to be able to read Luca QR codes by now.
(Obviously, that's still not the case, yet the feature is of great interest not only to me)
None of these sources seems to be the CWA web site -- suspicioulsly, the FAQ does not even mention Luca, let alone contains information about the status.

Please, add (und update) information about that to the FAQ, even if it is just "WIP" or "not going to happen".

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR adds a mention of the "luca" app to the FAQ entry `check_in_qr_code` as requested by @2019-05-10 in https://github.com/corona-warn-app/cwa-website/issues/1461#issuecomment-880800334. 

<details>
<summary>Preview of the changes</summary>

| English | German |
|---|---|
| Preview EN | Preview DE | 

</details>

---

This PR **closes #1461**.

### CODE REVIEW

**Comment by Ein-Tim** [comment]
I've touched unrelated files here, sorry. Will fix later.

**Comment by Ein-Tim** [comment]
Everything fixed, ready for review.

File: src/data/faq.json [hunk 1]
```diff
@@ -515,7 +515,7 @@
...
+                            "<ul><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>Who generates the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can the Event QR code also be set up and used for e.&nbsp;g. a private birthday party, providing this is legally possible?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>Where can the Event QR code be generated?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can Event QR codes from other apps (e.&nbsp;g. the 'luca' app) be scanned with the CWA?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Is there a way to check in without a smartphone (dongle or similar)?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>How should I scan the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> What happens if I scan an Event QR code but do not have the CWA installed on my smartphone?</a></li></ul>",
```
Comment (MikeMcC399) [inline src/data/faq.json:518]: There should be no space in English abbreviations: `e.&nbsp;g` is incorrect. 

https://de.wikipedia.org/wiki/Abk%C3%BCrzung#Schreibweise_und_Aussprache_nach_der_Rechtschreibreform_1996/2004/2006

"Wenn die Abkürzung mehrteilig ist, stehen im deutschen Sprachraum schmale Leerzeichen zwischen den Bestandteilen, z. B., d. h., i. d. R., ges. gesch., nicht jedoch im Englischen (e.g., a.m.).[7]"

File: src/data/faq.json [hunk 2]
```diff
@@ -595,7 +595,7 @@
                             "Yes of course, that is possible.",
                             "<b>Where can the Event QR code be generated?</b>",
                             "The QR code can be generated directly in the app or <a href='../eventregistration' target='_self' rel='noopener noreferrer'>here on the website</a>  and then printed as a PDF or scanned directly from the app.",
-                            "<b>Can Event QR codes from other apps be scanned with the CWA?</b>",
+                            "<b>Can Event QR codes from other apps (e.&nbsp;g. the 'luca' app) be scanned with the CWA?</b>",
```
Comment (MikeMcC399) [inline src/data/faq.json:598]: Also here, no `nbsp`

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
No space in English abbreviations.
Also according to American rules, there should be a "," after e.g. In British English writing there is no comma.

**Comment by Ein-Tim** [comment]
Thank you Mike! I did not know this and fixed this in https://github.com/corona-warn-app/cwa-website/pull/1500/commits/6526b70462a14205e6a81858dfc274ab6eb3ea5a.

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Done in https://github.com/corona-warn-app/cwa-website/commit/6526b70462a14205e6a81858dfc274ab6eb3ea5a.

File: src/data/faq.json (hunk 2 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Done in https://github.com/corona-warn-app/cwa-website/commit/6526b70462a14205e6a81858dfc274ab6eb3ea5a, too.

**Comment by heinezen** [comment]
@Ein-Tim If you want to revert changes in `HEAD`, you don't need an extra commit. You can either

* Amend the reverted changes to the previous commit (https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things)
* Do a soft reset and recommit the changes without the stuff you don't want (https://git-scm.com/docs/git-reset)

In this case, you should squash your first 3 commits to keep the repository cleaner :)

**Comment by Ein-Tim** [comment]
@heinezen 

Thank you. I'll read through the docs you shared and try to squash these commits here into one. If it doesn't work, I can also open a new PR with only one commit 🙂

Could you maybe clarify (internally) if this change is welcome?

**Comment by Ein-Tim** [comment]
@heinezen I tried to squash the commits but I'm really no git expert. I suggest to either use squash and merge here or I can offer to create a new PR, with only one commit. As you like.

**Comment by Ein-Tim** [comment]
PR #1505 is the correct PR, I feel like a noob now 😅

**Comment by MikeMcC399** [comment]
@Ein-Tim 
> I tried to squash the commits but I'm really no git expert. I suggest to either use squash and merge here or I can offer to create a new PR, with only one commit. 

I remember in the past trying to follow the instructions from @heinezen about squashing and I failed to complete it. I could squash locally but then GitHub wouldn't accept the push. I think it may be necessary to force push to GitHub after a local squash.

I question though if this is necessary when GitHub offers the "squash merging" function, which is very simple to use.
If there is concern about keeping the history of the repository clean, then perhaps squash merge should be made the default?
See https://docs.github.com/en/github/administering-a-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests.  That would need to be changed by a repository administrator.

**Comment by heinezen** [comment]
A squash is a rebase operation which requires a force push, yes. Force pushes are fine btw when the rebase succeeds locally.

> I question though if this is necessary when GitHub offers the "squash merging" function

Squash merge will squash all commit in the PR which is undesirable most of the time unless the changes are very small.

I would still like to encourage you to squash or amend your reverts whenever possible. Whenever you change a file in a commit, git saves a snapshot of the whole file, so the also repo size increases by the size of the involved files. Older commits are compressed, so it's not super significant for a single commit. But if you do it a lot, it adds up, especially when the files get larger. It also significantly improves readability of the history. If you ever contribute to other open source projects, most open source maintainers will also require that you keep the git history clean. It's better to learn the right way now.

You don't need to use the command line to squash. Any competent GUI git tool (maybe not Github Desktop) will allow you to squash commits from your local history with 2 or 3 clicks. They will also guide you through rebasing or any other "advanced" operation.

**Comment by MikeMcC399** [comment]
@heinezen 
Thanks for your hints and explanation.

Please consider adding your suggestions to https://github.com/corona-warn-app/cwa-website/blob/master/CONTRIBUTING.md if these should be used as a general rule for all contributors.

**Pull request closed without merging**